Packet++/header/SipLayer.h

```diff
@@ -69,6 +69,17 @@ namespace pcpp
 	class SipLayer : public TextBasedProtocolMessage
 	{
 	public:
+		/// SIP parse result types
+		enum class SipParseResult
+		{
+			/// Unknown or invalid format
+			Unknown = 0,
+			/// The message is a SIP request
+			Request = 1,
+			/// The message is a SIP response
+			Response = 2,
+		};
```

Comment:
This can be a private enum, no?

If we move it to be private, doxygen documentation is not required